src/Misc/LandingPage.js

```diff
@@ -0,0 +1,8 @@
+import React, { Component } from 'react';
+import LoginModal from '../LoginSignup/LoginModal';
```

Comment:
let do absolute import to make your life easierhttps://medium.com/@ktruong008/absolute-imports-with-create-react-app-4338fbca7e3d